src/chat/UnreadNotice.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { Animated, StyleSheet, Text, Easing } from 'react-native';
+import Icon from '../common/Icons';
+
+const styles = StyleSheet.create({
+  unreadContainer: {
+    padding: 2,
+    backgroundColor: '#96A3F9',
+    flexDirection: 'row',
+    position: 'absolute',
+    left: 0,
+    right: 0,
+    zIndex: 1,
+    alignItems: 'center'
+  },
+  unreadText: {
+    flex: 0.9,
+    color: '#FFFFFF',
+    fontSize: 14
+  },
+  icon: {
+    flex: 0.05,
+    width: 14,
+    height: 14,
+    margin: 8,
+    fontSize: 14,
+    color: 'white',
+    textAlign: 'center',
+  }
+  /*
+  crossIcon: {
+    fontSize: 16,
+    fontWeight: 'bold',
+    transform: [
+      { rotate: '45deg' }
+    ]
+  }
+  */
+});
+
+const POSITIONS = {
+  top: 'top',
+  bottom: 'bottom'
+};
+
+const showAnimationConfig = {
+  toValue: 1,
+  duration: 400,
+  easing: Easing.bezier(0.17, 0.67, 0.11, 0.99)
+};
+
+const hideAnimationConfig = {
+  toValue: 0,
+  duration: 400,
+  easing: Easing.bezier(0.17, 0.67, 0.11, 0.99)
+};
+
+export default class UnreadNotice extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      translateAnim: new Animated.Value(0)
```

Comment:
You can just use `translateAnimated` and then will not need to add `Anim` to the dict. It is still a reasonable name.